src/data.jl

```diff
@@ -303,12 +351,36 @@ Electrum.PeriodicAtomList(s::Supercell) = s.atomlist
 Base.getindex(s::Supercell, i...) = getindex(s.atomlist, i...)
 
 mutable struct raMOStatus
```

Comment:
This is just a general note: if you have a struct with a `const` array (or any other mutable type), the object can still be mutated by property access (this is why dot syntax is discouraged - you're reaching into internals that should be hidden): what's actually constant is the reference to the array (implemented as a kind of pointer).